src/Tests/AkamaiCacheControlFormTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * Simpletest test for Akamai cache control form clearing tests.
+ */
+
+namespace Drupal\akamai\Tests;
+
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Test the Akamai Homepage Clearing.
+ *
+ * @description Test Akamai cache clearings of the site homepage.
+ *
+ * @group Akamai
+ */
+class AkamaiCacheControlFormTest extends WebTestBase {
+
+  /**
+   * Node created.
+   */
+  protected $node;
+
+  /**
+   * User with admin rights.
+   */
+  protected $privilegedUser;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = ['system_test', 'node', 'user', 'akamai'];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    // Create and log in our privileged user.
+    $this->privilegedUser = $this->drupalCreateUser(array(
+      'purge akamai cache',
+      'administer akamai',
+      'purge akamai cache',
+    ));
+    $this->drupalLogin($this->privilegedUser);
+    $this->drupalCreateContentType(['type' => 'article']);
+    $this->node = $this->drupalCreateNode(['type' => 'article']);
+  }
+
+  /**
+   * Tests that manual Akamai Cache Clear page.
+   */
+  public function testValidUrlPurging() {
+
+    $edit['paths'] = 'node/1';
+    $edit['domain_override'] = 'staging';
+    $edit['action'] = 'invalidate';
+
+    $this->drupalPostForm('admin/config/development/performance/akamai-cache-clear', $edit, t('Start Refreshing Content'));
+    $this->assertText(t('Requested invalidate of the following URLs: /node/1'), t('node/1 URLs purged'));
```

Comment:
IMO merge both methods into single one and first do postitive testing and then negative testing in this way test don't need to instantiate the environment again.